src/main/java/io/jenkins/pluginhealth/scoring/probes/UpdateCenterPluginPublicationProbe.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Jenkins Infra
```

Comment:
This needs to be the year of the file creation.

```suggestion
 * Copyright (c) 2023 Jenkins Infra
```